src/SechsPunktVier.java

```diff
@@ -0,0 +1,30 @@
+public class SechsPunktVier {
+    public static void insertionSort(int array[]) {
```

Comment:
Teamwork makes the dream work 🤔 es bringt euch nicht viel Übungen abzuschreiben, es ist super wenn ihr euch wo hilfe holt und zusammenarbeitet. 

Wenn ihr bei einer übung aber was nicht versteht und das einfach "irgendwie" löst dann fragt bitte nach damit wir die Lösung gemeinsam durchbesprechen können,  😉